CHANGELOG.md

```diff
@@ -24,6 +24,7 @@
 
 ### Improved
 
+- Fixed issue with react hot loader for IE
```

Comment:
Can you move this to `master`? Version `7.1.0` is already released!

Reply:
oops, good catch 👍 
Done